### PR TITLE
[COM-138] Select component implementation

### PR DESCRIPTION
**Pull request by Angele91**
## Please delete checklist items and sections that are not relevant

**Checklist**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**Please check if the PR fulfills these requirements**

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Storybook or docs have been added / updated (for bug fixes / features)

**What is the ticket / issue associated with this change?** (Link to the ticket or issue)

https://impulsumvc.atlassian.net/browse/COM-138

**Storybook Link** (Link to a local storybook url to review new functionality)

http://localhost:8080/?path=/story/components-primitives-select--all

### CODE REVIEW

File: packages/components/src/components/primitives/Select/Select.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    },
+  },
+} as Meta;
+
+const AllTemplate = () => {
+  const [values, setValues] = React.useState({});
+
+  return (
+    <VStack divider={<StackDivider borderColor="gray.200" />} spacing={4}>
+      {selectSizes.map((size) => (
```
Comment (tkanzakic) [inline packages/components/src/components/primitives/Select/Select.stories.tsx:26]: please show the size label in the story

File: packages/components/src/components/primitives/Select/Select.test.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import TestRenderer from 'react-test-renderer';
+import { renderWithProviders } from 'tests/renderWithProviders';
+import { ComposerProvider } from 'theme';
+import { Select } from '.';
+
+describe('Select', () => {
+  test('it should render', () => {
+    renderWithProviders(<Select />);
```
Comment (tkanzakic) [inline packages/components/src/components/primitives/Select/Select.test.tsx:9]: can we assert that the component is rendered?

**Review by tkanzakic — APPROVED** [review]
small comments and LGTM

File: packages/components/src/components/primitives/Select/Select.stories.tsx (hunk 1 above)
Comment (Angele91) [reply packages/components/src/components/primitives/Select/Select.stories.tsx:33]: Done!

File: packages/components/src/components/primitives/Select/Select.test.tsx (hunk 2 above)
Comment (Angele91) [reply packages/components/src/components/primitives/Select/Select.test.tsx:10]: Done!

**Pull request merged**